aspnetcore/mvc/views/razor.md

```diff
@@ -599,6 +630,46 @@ If multiple import files have the `@namespace` directive, the file closest to th
 
 The `@section` directive is used in conjunction with the [layout](xref:mvc/views/layout) to enable pages or views to render content in different parts of the HTML page. For more information, see [Sections](xref:mvc/views/layout#layout-sections-label).
 
+<a name="using"></a>
+
+### @using
+
+The `@using` directive adds the C# `using` directive to the generated view:
+
+[!code-cshtml[](razor/sample/Views/Home/Contact9.cshtml)]
+
+::: moniker range=">= aspnetcore-3.0"
+
+## Directive attributes
```

Comment:
Directive attributes are more than what's here. I'd take these out of the Razor directives file and make them their own thing since there's a whole bunch of specific applications `@bind` etc.